js/index.js

```diff
@@ -1,41 +1,86 @@
 import { Game } from './game.js';
 import { Player } from './player.js';
-import {get, getAll} from './util/utilities.js'
+import {get, getAll, createEl} from './util/utilities.js'
 
-
-//Notionally from DOM
-let playerOneName = 'Travis';
-let playerTwoName = 'TRobot';
-let playerThreeName = 'TJustins';
-const boardFromDOM = document.querySelectorAll('.game-board');
 let game;
+const boardFromDOM = document.querySelectorAll('.game-board');
+const domMethods = {
+  render
+}
 
-//create users array
-get('#start-btn').addEventListener('click', () => {
-  // playerOneName = get('#p1-name-input').value;
-  // playerTwoName = get('#p2-name-input').value;
-  // playerThreeName = get('#p3-name-input').value;
-
-  //set screen transition
-    //add transition img
-  get('#view').innerHTML = '<img src="./img/loading.gif">';
+get('#start-btn').addEventListener('click',transitionToGame );
+get('#view').addEventListener('click', render);
```

Comment:
Not a bad idea to have helper methods to clean up your code a bit, but this is exactly what jQuery is for. You'll run into some huge issues down the road trying to use a simple word like `get` to represent retrieving a DOM element. `get` is actually a reserved keyword in some scenarios and a common word we'll be using when you start doing network requests. I would either ditch this for plain jQuery/JavaScript, or change your method names to something a bit more specific.